src/connections/networks/index.ts

```diff
@@ -4,7 +4,8 @@ import { RelayChain } from '../../services/crowdloan/types'
 
 export const subsocial = {
   name: 'Subsocial',
-  node: 'wss://para.subsocial.network',
+  node: 'https://rco-para.subsocial.network',
+  wsNode: 'wss://rco-para.subsocial.network',
```

Comment:
this one should be changed back to subsocial before merge right?

Reply:
Yes